packages/py3dns/PKGBUILD

```diff
@@ -24,5 +24,7 @@ package() {
   cd "$_pkgname-$pkgver"
 
   python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+  
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$_pkgname-$pkgver/LICENSE"
```

Comment:
`install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"` do not include the version